src/ray/raylet/scheduling_policy.cc

```diff
@@ -39,6 +39,11 @@ std::unordered_map<TaskID, ClientID> SchedulingPolicy::Schedule(
       decision[task_id] = local_client_id;
       continue;
     }
+    // Place actor tasks locally.
+    if (t.GetTaskSpecification().IsActorTask()) {
```

Comment:
yes, it's a fact, actually. Remote actor tasks are never enqueued locally. This is handled in `NodeManager::SubmitTask`. An actor task is enqueued as placeable here:
https://github.com/ray-project/ray/blob/master/src/ray/raylet/node_manager.cc#L651
if the actor is local, and forwarded here:
https://github.com/ray-project/ray/blob/master/src/ray/raylet/node_manager.cc#L656
if the actor is remote.

Reply:
So the actor check you mentioned already happened by the time `SchedulingPolicy::Schedule()` runs.

Reply:
Ok, it's probably worth including a CHECK statement here about that since changes in `SubmitTask` or `Schedule` could potentially break this invariant.

Reply:
@atumanov Let's add the check or add a comment.